web/src/pages/Main.tsx

```diff
@@ -37,10 +37,8 @@ export const MainPage = (props: { mercuryApi: MercuryAPI }): JSX.Element => {
   const [componentProperties, setComponentProperties] =
     useState<TComponentProperties>()
   const [isLoading, setIsLoading] = useState<boolean>(true)
-  const [feedFlow, setFeedFlow] = useState<TFeedFlow>({
```

Comment:
Does that mean we can remove the TFeedFlow type?

Reply:
Yes. We assume the feed flow is cubic everywhere and just convert/display it as mass flow if the switch is clicked.